warp/yul/main.py

```diff
@@ -45,8 +59,11 @@ def transpile_from_solidity(
             check=True,
             capture_output=True,
         )
+        if result.stderr:
+            print(bcolors.FAIL + bcolors.BOLD + result.stderr.decode("utf-8"))
+            sys.exit(1)
     except subprocess.CalledProcessError as e:
-        print(e.stderr.decode("utf-8"), file=sys.stderr)
+        print(bcolors.FAIL + bcolors.BOLD + e.stderr.decode("utf-8"), file=sys.stderr)
```

Comment:
I am not an expert in ansi escape sequences, but don't they need a closing one? Otherwise, everything afterwards is also colored

Reply:
Added closing (`bcolors.ENDC`)